web/frontend/src/components/slam-2d-render.vue

```diff
@@ -295,20 +223,22 @@ const updatePointCloud = (pointcloud: Uint8Array) => {
   const viewWidth = viewHeight * 2;
 
   const points = loader.parse(pointcloud.buffer);
+  pointsMaterial = points.material as THREE.PointsMaterial;
+  pointsMaterial.sizeAttenuation = false;
+  pointsMaterial.size = 4;
   points.geometry.computeBoundingSphere();
+  points.renderOrder = pointsRenderOrder;
 
   const { radius = 1, center = { x: 0, y: 0 } } = points.geometry.boundingSphere ?? {};
   camera.position.set(center.x, center.y, 100);
+  controls.target.set(center.x, center.y, 0);
   camera.lookAt(center.x, center.y, 0);
 
   const aspect = canvas.clientHeight / canvas.clientWidth;
   camera.zoom = aspect > 1
-    ? viewHeight / (radius * 2)
-    : camera.zoom = viewWidth / (radius * 2);
+    ? viewHeight / (radius * 4)
```

Comment:
1. why * 4?
2. Can this please be added to a constant at the top of the file?

Reply:
It's approximately `1/aspect`. The equation isn't great right now, I'll add a follow up with one that's more comprehensive for a variety of cases that aren't handled yet. For now I've added a constant for it.